src/main/java/org/mockito/internal/creation/instance/ConstructorInstantiator.java

```diff
@@ -107,10 +107,7 @@ private String constructorArgsString() {
     private InstantiationException multipleMatchingConstructors(Class<?> cls, List<Constructor<?>> constructors) {
         return new InstantiationException(join("Unable to create instance of '" + cls.getSimpleName() + "'.",
                 "Multiple constructors could be matched to arguments of types " + constructorArgTypes() + ":",
-                join("", " - ", constructors),
-                "If you believe that Mockito could do a better join deciding on which constructor to use, please let us know.",
-                "Ticket 685 contains the discussion and a workaround for ambiguous constructors using inner class.",
-                "See https://github.com/mockito/mockito/issues/685"
+                join("", " - ", constructors)
```

Comment:
Let's keep the extra information. I think it is useful to:
 - inform about the inner class pattern for working around the ambiguous constructors
 - ask for feedback - it does not hurt to do that

You can keep the original or modify it if you want. I'd like to keep at least the hint about inner class.